tests/cpu/Config_tests.cpp

```diff
@@ -4922,6 +4944,189 @@ OCIO_ADD_TEST(Config, fixed_function_serialization)
+    }
+
+    {
+        const std::string strEnd =
+            "    from_scene_reference: !<GroupTransform>\n"
+            "      children:\n"
+            "        - !<FixedFunctionTransform> {style: ACES2_RGB_TO_JMh, params: [100, 0.64, 0.33, 0.3, 0.6, 0.15, 0.06, 0.3127, 0.329]}\n"
+            "        - !<FixedFunctionTransform> {style: ACES2_RGB_TO_JMh, params: [100, 0.64, 0.33, 0.3, 0.6, 0.15, 0.06, 0.3127, 0.329], direction: inverse}\n";
+
+        const std::string str = PROFILE_V21_START + strEnd;
+
+        OCIO::LogGuard log; // Mute the experimental warnings.
+
+        std::istringstream is;
+        is.str(str);
+
+        OCIO::ConstConfigRcPtr config;
+        OCIO_CHECK_THROW_WHAT(config = OCIO::Config::CreateFromStream(is), OCIO::Exception,
+            "Only config version 2.4 (or higher) can have FixedFunctionTransform style 'ACES_RGB_TO_JMH_20'.");
+    }
+
+    {
+        const std::string strEnd =
+            "    from_scene_reference: !<GroupTransform>\n"
+            "      children:\n"
+            "        - !<FixedFunctionTransform> {style: ACES2_TonescaleCompress, params: [100, 0.64, 0.33, 0.3, 0.6, 0.15, 0.06, 0.3127, 0.329]}\n"
+            "        - !<FixedFunctionTransform> {style: ACES2_TonescaleCompress, params: [100, 0.64, 0.33, 0.3, 0.6, 0.15, 0.06, 0.3127, 0.329], direction: inverse}\n";
+
+        const std::string str = PROFILE_V21_START + strEnd;
+
+        OCIO::LogGuard log; // Mute the experimental warnings.
+
+        std::istringstream is;
+        is.str(str);
+
+        OCIO::ConstConfigRcPtr config;
+        OCIO_CHECK_THROW_WHAT(config = OCIO::Config::CreateFromStream(is), OCIO::Exception,
+            "Only config version 2.4 (or higher) can have FixedFunctionTransform style 'ACES_TONESCALE_COMPRESS_20'.");
+    }
+
+    {
+        const std::string strEnd =
+            "    from_scene_reference: !<GroupTransform>\n"
+            "      children:\n"
+            "        - !<FixedFunctionTransform> {style: ACES2_GamutCompress, params: [100, 0.64, 0.33, 0.3, 0.6, 0.15, 0.06, 0.3127, 0.329]}\n"
+            "        - !<FixedFunctionTransform> {style: ACES2_GamutCompress, params: [100, 0.64, 0.33, 0.3, 0.6, 0.15, 0.06, 0.3127, 0.329], direction: inverse}\n";
+
+        const std::string str = PROFILE_V21_START + strEnd;
+
+        OCIO::LogGuard log; // Mute the experimental warnings.
+
+        std::istringstream is;
+        is.str(str);
+
+        OCIO::ConstConfigRcPtr config;
+        OCIO_CHECK_THROW_WHAT(config = OCIO::Config::CreateFromStream(is), OCIO::Exception,
+            "Only config version 2.4 (or higher) can have FixedFunctionTransform style 'ACES_GAMUT_COMPRESS_20'.");
+    }
+
+    {
+        const std::string strEnd =
+            "    from_scene_reference: !<GroupTransform>\n"
+            "      children:\n"
+            "        - !<FixedFunctionTransform> {style: ACES2_OutputTransform, params: []}\n";
```

Comment:
This triggered a crash on the Windows CI, which I couldn't replicate anywhere including my own Windows workstation. But clang sanitizer caught the issue:
```
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk/usr/include/c++/v1/vector:1538:12: runtime error: reference binding to null pointer of type 'double'
SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk/usr/include/c++/v1/vector:1538:12 in 
```

Added a check in [OCIOYaml.cpp](https://github.com/AcademySoftwareFoundation/OpenColorIO/pull/1983/files#diff-efc8a26ff824080bff3b85e2605aa47b120d2fa14b0e8a586b447c6405b3a62dR1354).